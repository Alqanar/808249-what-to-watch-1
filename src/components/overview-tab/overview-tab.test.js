import React from "react";
import renderer from "react-test-renderer";

import OverviewTab from "./overview-tab.tsx";


const filmMockData = {
  rating: 8.8,
  scoresCount: 441946,
  description: `After the devastating events of Avengers: Infinity War (2018), the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to undo Thanos' actions and restore order to the universe.`,
  director: `Russo brothers`,
  starring: [`Robert Downey Jr.`, `Chris Hemsworth`, `Mark Ruffalo`]
};

it(`Overview tab is correctly renders`, () => {
  const tree = renderer
    .create(
      <OverviewTab
        film={filmMockData}
      />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
