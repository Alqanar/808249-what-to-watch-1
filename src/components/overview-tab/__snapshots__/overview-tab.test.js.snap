// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview tab is correctly renders 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8.8
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        441946 ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      After the devastating events of Avengers: Infinity War (2018), the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to undo Thanos' actions and restore order to the universe.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: Russo brothers
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: Robert Downey Jr., Chris Hemsworth, Mark Ruffalo
      </strong>
    </p>
  </div>,
]
`;
